bin/check-process.rb

```diff
@@ -287,7 +300,7 @@ def run
 
     if config[:metric]
       # #YELLOW
-      count = procs.map { |p| p[config[:metric]].to_i }.reduce { |a, b| a + b } # rubocop:disable SingleLineBlockParams
+      count = procs.map { |p| p[config[:metric]].to_i }.reduce { |a, e| a + e }
```

Comment:
I don't see a real change so unless someone can explain why one is good and the other is bad I am going to assume that this is a false positive, highly subjective and opinionated, or just a bad rule and would prefer to ignore behavior rather than make a change here. Would you be open to creating an issue on rubocop and feel free to tag me on this? If there is something that makes this special I would like to know.